tests/vm_test.go

```diff
@@ -1724,6 +1724,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 					}, 360*time.Second, 1*time.Second).Should(BeTrue())
 				})
 				DescribeTable("with a failing VMI and the kubevirt.io/keep-launcher-alive-after-failure annotation", func(keepLauncher string) {
+					var launcherPod *k8sv1.Pod
```

Comment:
Please declare it as close as possible to the usage.

Reply:
Done